iset.mm

```diff
@@ -68631,6 +68632,25 @@ property of disjoint unions (see ~ updjud in the case of functions).
       enumct simplbiim ) AGHCDIJDAKCAKELAFLMFNEOPOAQRBLMBNCDAFESAFBETUA $.
   $}
 
+  ${
+    $d f x $.
+    $( ` _om ` is countable.  (Contributed by Jim Kingdon, 23-Dec-2023.) $)
+    omct $p |- E. f f : _om -onto-> ( _om |_| 1o ) $=
+      ( vx com c1o cdju cv wfo wex cid cres wf1o f1oi f1ofo cvv wcel omex ax-mp
+      resiexg mp2b c0 foeq1 spcev wb peano1 elex2 ctm mpbir ) CCDEAFZGAHZCCUHGZ
+      AHZCCICJZKCCULGZUKCLCCULMUJUMAULCNOULNOPCNRQCCUHULUAUBSTCOBFCOBHUIUKUCUDB
+      TCUEBCAUFSUG $.
+  $}
+
+  ${
+    $d A f $.
+    $( A countable class is a set.  (Contributed by Jim Kingdon,
+       25-Dec-2023.) $)
```

Comment:
Maybe a back-link to ctex ?

Reply:
I've been back and forth on this a little bit, both before and after your comment.

The case against a link to ctex is that `A ~<_ _om` is a dead end constructively and there's no point in drawing attention to it when people have landed on `E. f f : _om -onto-> ( A |_| 1o )` .

The case for is, well, why not it is just a link.

I guess for now I'll merge without the link but we can always add it if we decide we want it later.